Scripts/06 SL-Utilities.lua

```diff
@@ -68,7 +68,7 @@ end
 -- in seconds, for how long you want the text to appear on screen.
 -- in Simply Love, the default SystemMessage duration used in ./BGA/ScreenSystemLayer overlay.lua is 3
 
-SM = function( arg, duration )
+SM = function( arg, stack, duration )
```

Comment:
Move stack to the end to make sure we don't break any backwards compatibility for the function call itself. I know it's handled above but it's better to do so.